internal/pkg/agent/application/actions/handlers/handler_action_policy_change_test.go

```diff
@@ -112,7 +112,7 @@ func TestPolicyChangeHandler_handlePolicyChange_FleetClientSettings(t *testing.T
 		}))
 
 	fleetServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if r.URL.Path != "api/status" {
+		if r.URL.Path != "/api/status" {
```

Comment:
This looks like it might have been done in support of a test, but I don't see a new test or any new code to hit this path and get a NotFound back in the diff.